### PR TITLE
Fix invoices & donations alignment #6742

### PR DESCRIPTION
**Pull request by manuthecoder**
See #6742

### CODE REVIEW

**Comment by sampoder** [comment]
Can you please attach screenshots to all of these PRs?

**Comment by manu-gurudath** [comment]
Sure! Here you go:

image
image


(red markup is the changes I made)

File: app/views/events/reimbursements.html.erb [hunk 1]
```diff
@@ -41,20 +41,21 @@
```
Comment (sampoder) [inline app/views/events/reimbursements.html.erb]: I think this is unrelated?

**Review by sampoder — DISMISSED** [review]
LGTM!
